src/main/java/org/electronicReferences/specification/UserSpecification.java

```diff
@@ -0,0 +1,13 @@
+package org.electronicReferences.specification;
+
+import org.electronicReferences.models.User;
+import org.springframework.stereotype.Component;
+import org.springframework.data.jpa.domain.Specification;
+
+@Component
+public class UserSpecification {
```

Comment:
Are you going to have multiple specifications?

Reply:
No, only this one